src/api/spec/models/branch_package_spec.rb

```diff
@@ -0,0 +1,31 @@
+require 'rails_helper'
+
+RSpec.describe BranchPackage, vcr: true do
+  let(:user) { create(:confirmed_user, login: 'tom') }
+  let(:home_project) { user.home_project }
+  let!(:project) { create(:project, name: 'BaseDistro') }
+  let!(:package) { create(:package, name: 'test_package', project: project) }
+
+  context '#branch' do
+    let(:branch) { BranchPackage.new(project: project.name, package: package.name).branch }
```

Comment:
BranchPackage is a helper, so not sure if we should introduce a factory for it atm

``
src/api/app/helpers/branch_package.rb
``